var/spack/repos/builtin/packages/rosco/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Rosco(CMakePackage):
+    """
+    ROSCO controls package for OpenFAST from NREL
+    Note: this only builds ROSCO controls library for inclusion with OpenFAST
+    If the toolbox or tuning scripts are needed, please build manually
+    """
+
+    homepage = "https://rosco.readthedocs.io/en/latest/"
+    url = "https://github.com/NREL/ROSCO/archive/refs/tags/v2.9.0.tar.gz"
+    git = "https://github.com/NREL/ROSCO.git"
+
+    maintainers("dzalkind", "ndevelder")
```

Comment:
We need to confirm willingness to be maintainers.  Sounds like @ndevelder is in.  @dzalkind please let @marchdf know if you want to be listed as a maintainer for this package.

Reply:
Sure, I am willing.